releasenotes/notes/botocore-exclude-records--74ac3e5d4d4fdc95.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    The Records parameter to Firehose endpoint calls is being excluded from the
+    tags to avoid generating traces with a large payload.
+
```

Comment:
```suggestion
    The ``Records`` parameter to ``Firehose`` endpoint calls is being excluded from the
    tags to avoid generating traces with a large payload.

```